core/src/main/kotlin/com/malinskiy/marathon/execution/queue/QueueActor.kt

```diff
@@ -50,6 +50,7 @@ class QueueActor(
 
     init {
         queue.addAll(testShard.tests + testShard.flakyTests)
+        progressReporter.init(configuration)
         progressReporter.totalTests(poolId, queue.size)
```

Comment:
Maybe it make sense to add `configuration` as parameter to `totalTests` func

Reply:
Removed as well.